docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -71,7 +71,7 @@ to generate the `extraData` RLP string to include in the genesis file.
     besu rlp encode --from=toEncode.json
     ```    
 
-Where the `toEncode.json` file contains a list of the initial validators in ascending order. 
+Where the `toEncode.json` file contains a list of the initial validators in ascending order. Use the Besu subcommand [`public-key export-address`](../../../Reference/CLI/CLI-Subcommands.md#export-address) to write the address of a validator and copy it to the `toEncode.json` file. For example:
 
 !!! example "One Initial Validator"
```

Comment:
Now the extra sentence has been added about the `public-key export-address` command it's not clear the example is the `toEncode.json` file rather than the `public-key export-address` command output.  Suggest updating the title to _One Initial Validator in toEncode.json File_

Reply:
Updated.